src/Raven.Server/Documents/TombstoneCleaner.cs

```diff
@@ -158,6 +158,8 @@ internal class DeleteTombstonesCommand : TransactionOperationsMerger.MergedTrans
 
             public long NumberOfTombstonesDeleted { get; private set; }
 
+            public override bool UpdateAccessTime => false;
```

Comment:
Can we do this a regular field and we set it? That avoids the virtual method call at all time

Reply:
done: https://github.com/ravendb/ravendb/pull/11272